BadHabbit/BadHabbit/UserStories/MainTabBar/MainTabBarAssembly.swift

```diff
@@ -19,17 +19,29 @@ final class MainTabBarAssembly {
     
     func assemble() -> UIViewController {
         let tabVC = UITabBarController()
-        
+        tabVC.tabBar.backgroundColor = .white
```

Comment:
why?

Reply:
I changed unselectedItem color instead of tabBar color.  If we do not change the color, the inactive tabBarItem is not visible